src/modules/coreHttpApi/openapi.yml

```diff
@@ -803,6 +803,106 @@ paths:
         403:
           $ref: "#/components/responses/Forbidden"
 
+  /api/v2/Attributes/ExecuteIQLQuery:
+    post:
+      operationId: executeIQLQuery
+      description: Execute IQL Query
+      tags:
+        - Attributes
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                query:
+                  type: object
+                  properties:
+                    queryString:
+                      description: IQL Query String
+                      type: string
+                      example: PhoneNumber
```

Comment:
is `PhoneNumber` a correct example for an IQL Query String?

Reply:
I know (now) that PhoneNumber is a correct example, but please do not resolve conversations without commenting.

And please give a bigger example for a query string here.